.github/workflows/lint-and-unit.yml

```diff
@@ -0,0 +1,55 @@
+name: Tox Lint and Unit Tests
+
+on:
+  push:
```

Comment:
I'm not very familiar with workflows, but doesn't this line mean the jobs defined in this file should've triggered for this PR and pushed commits to it? Or do you need to add something to get it to show up in the Checks tab?

Reply:
Currently, this triggers on my personal fork, but yes I think it will start working when we merge it on push / on PR. This is setup the same way as we do in the main armada repo, see here.

https://github.com/G-Research/armada/blob/master/.github/workflows/python-client.yml